GOVERNANCE.md

```diff
@@ -41,14 +41,12 @@ Determination of the complete criteria for contributors is the responsibility of
 
 Contributors are eligible to vote on Technical Committee elections. Contributors do not have the rights to accept pull requests.
 
-Being contributor is a prerequisite for becoming a maintainer to the projects they are contributing to.
-In order for a contributor to become a maintainer, the individual must be nominated by one of the existing maintainers with a reference to the individual's history of contributions to the project.
-The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
-
 ### Maintainers
 
 Maintainers are active contributors and participants in the projects they are maintainers of. Maintainers have the rights to accept pull requests.
 
+In order for a contributor to become a maintainer for a certain project, the individual must be nominated by one of the existing maintainers of that project with a reference to the individual's history of contributions to the same project. The nomination is discussed and voted on amongst the maintainers. Unless unanimous approval within the maintainers group can be reached, the Technical Committee can be consulted for advice. If approved, the nominated contributor must be added to the maintainer list.
```

Comment:
Have we defined the "maintainer list" somewhere?

Reply:
I don't think so. I'm not sure it needs to be defined, but we could add a sentence to the top paragraph and reference the GH teams. However, I don't know what status you must have to be able to inspect the members of a team, though (or even see what team controls the maintainership of a git). I reckon you must at least be a member of the organization?

Reply:
What about this instead?
```suggestion
In order for a contributor to become a maintainer for a certain project, the individual must be nominated by one of the existing maintainers of that project with a reference to the individual's history of contributions to the same project. The nomination is discussed and voted on amongst the maintainers. Unless unanimous approval within the maintainers group can be reached, the Technical Committee can be consulted for advice. If approved, the nominated contributor must be added to the GitHub team for the maintainers of this project.
```